--- clamav-0.93rc1/configure.in~	2008-03-30 15:25:01.112614075 +0200
+++ clamav-0.93rc1/configure.in	2008-03-30 15:26:27.256732255 +0200
@@ -472,9 +472,9 @@
     sendmailver_b=`echo $sendmailver | awk -F. '{printf $2}'`
     sendmailver_c=`echo $sendmailver | awk -F. '{printf $3}'`
 
-    AC_DEFINE_UNQUOTED([SENDMAIL_VERSION_A], $sendmailver_a, [major version of Sendmail])
-    AC_DEFINE_UNQUOTED([SENDMAIL_VERSION_B], $sendmailver_b, [minor version of Sendmail])
-    AC_DEFINE_UNQUOTED([SENDMAIL_VERSION_C], $sendmailver_c, [subversion of Sendmail])
+    AC_DEFINE(SENDMAIL_VERSION_A, $sendmailver_a, [major version of Sendmail])
+    AC_DEFINE(SENDMAIL_VERSION_B, $sendmailver_b, [minor version of Sendmail])
+    AC_DEFINE(SENDMAIL_VERSION_C, $sendmailver_c, [subversion of Sendmail])
   fi
 fi
 
