diff -urN clamav-0.65.orig/etc/clamav.conf clamav-0.65/etc/clamav.conf
--- clamav-0.65.orig/etc/clamav.conf	Mon Dec  1 14:59:35 2003
+++ clamav-0.65/etc/clamav.conf	Mon Dec  1 14:57:07 2003
@@ -5,12 +5,12 @@
  
 
 # Comment or remove the line below.
-Example
+# Example
 
 # Uncomment this option to enable logging.
 # LogFile must be writable for the user running the daemon.
 # Full path is required.
-#LogFile /tmp/clamd.log
+#LogFile /var/log/clamd.log
 
 # By default the log file is locked for writing - the lock protects against
 # running clamd multiple times (if want to run another clamd, please
@@ -30,19 +30,20 @@
 #LogTime
 
 # Use system logger (can work together with LogFile).
-#LogSyslog
+LogSyslog
 
 # Enable verbose logging.
-#LogVerbose
+# LogVerbose
 
 # This option allows you to save the process identifier of the listening
 # daemon (main thread).
-#PidFile /var/run/clamd.pid
+# We use dir becouse covering clamd.pid file couse default rights are sane.
+PidFile /var/run/clamav/clamd.pid 
 
 # Path to a directory containing .db files.
 # Default is the hardcoded directory (mostly /usr/local/share/clamav,
 # it depends on installation options).
-#DataDirectory /var/lib/clamav
+DataDirectory /var/lib/clamav
  
 # The daemon works in local or network mode. Currently the local mode is
 # recommended for security reasons.
@@ -50,10 +51,12 @@
 # Path to the local socket. The daemon doesn't change the mode of the
 # created file (portability reasons). You may want to create it in a directory
 # which is only accessible for a user running daemon.
-LocalSocket /tmp/clamd
+# LocalSocket /var/run/clamav/clamd.socket
+
+LocalSocket /var/lib/clamav/clamd.socket
 
 # Remove stale socket after unclean shutdown.
-#FixStaleSocket
+FixStaleSocket
 
 # TCP port address.
 #TCPSocket 3310
@@ -111,11 +114,11 @@
 
 # Run as selected user (clamd must be started by root).
 # By default it doesn't drop privileges.
-#User clamav
+User clamav
 
 # Initialize the supplementary group access (for all groups in /etc/group
 # user is added in. clamd must be started by root).
-#AllowSupplementaryGroups
+AllowSupplementaryGroups
 
 # Don't fork into background. Useful in debugging.
 #Foreground
@@ -128,13 +131,12 @@
 ##
 
 # Uncomment this option if you are planning to scan mail files.
-#ScanMail
+ScanMail
 
 ##
 ## Archive support
 ##
 
-
 # Comment this line to disable scanning of the archives.
 ScanArchive
 
