diff -uNr clamav-0.80rc4.orig/configure clamav-0.80rc4/configure
--- clamav-0.80rc4.orig/configure	2004-10-11 04:20:10.000000000 +0200
+++ clamav-0.80rc4/configure	2004-10-16 23:35:11.146407944 +0200
@@ -1042,7 +1042,7 @@
   --with-dbdir=path	  path to virus database directory
   --with-db1=name	  name of the main database (default=main.cvd)
   --with-db2=name	  name of the daily database (default=daily.cvd)
-  --with-tcpwrappers      support hosts.allow / hosts.deny (default=auto)
+  --with-tcpwrappers      support hosts.allow / hosts.deny (default=no)
 
 Some influential environment variables:
   CC          C compiler command
@@ -12176,7 +12176,7 @@
    { (exit 1); exit 1; }; } ;;
   esac
 else
-   tcpw=auto
+   tcpw=no
 fi;
 
 if test "$tcpw" != "no" && test "$have_milter" = "yes" ; then
diff -uNr clamav-0.80rc4.orig/configure.in clamav-0.80rc4/configure.in
--- clamav-0.80rc4.orig/configure.in	2004-10-11 04:13:35.000000000 +0200
+++ clamav-0.80rc4/configure.in	2004-10-16 23:33:12.456451560 +0200
@@ -480,12 +480,12 @@
 dnl tcpwrappers support
 dnl rules from http://ma.ph-freiburg.de/tng/tng-technical/2002-01/msg00094.html
 AC_ARG_WITH(tcpwrappers,
-[  --with-tcpwrappers      support hosts.allow / hosts.deny (default=auto)],
+[  --with-tcpwrappers      support hosts.allow / hosts.deny (default=no)],
 [ case "$withval" in
   yes|no) tcpw="$withval" ;;
   *)      AC_MSG_ERROR([--with-tcpwrappers does not take an argument]) ;;
   esac],
-[ tcpw=auto ])
+[ tcpw=no ])
 
 if test "$tcpw" != "no" && test "$have_milter" = "yes" ; then
         AC_CHECK_HEADERS(tcpd.h,[
