--- clamav-0.98.1/freshclam/notify.c.org	2014-03-15 12:30:42.386370064 +0100
+++ clamav-0.98.1/freshclam/notify.c	2014-03-15 12:37:47.097443450 +0100
@@ -82,8 +82,8 @@
 
         if ((sockd = socket (AF_UNIX, SOCK_STREAM, 0)) < 0)
         {
-            perror ("socket()");
-            logg ("^Clamd was NOT notified: Can't create socket endpoint for %s\n", opt->strarg);
+            logg ("^Clamd was NOT notified: Can't create socket endpoint for %s: %s\n",
+                opt->strarg, strerror(errno));
             optfree (opts);
             return -1;
         }
@@ -92,9 +92,9 @@
             (sockd, (struct sockaddr *) &server,
              sizeof (struct sockaddr_un)) < 0)
         {
-            perror ("connect()");
+            logg ("^Clamd was NOT notified: Can't connect to clamd through %s: %s\n",
+                opt->strarg, strerror(errno));
             closesocket (sockd);
-            logg ("^Clamd was NOT notified: Can't connect to clamd through %s\n", opt->strarg);
             optfree (opts);
             return -11;
         }
@@ -137,8 +137,7 @@
 
         if ((sockd = socket (res->ai_family, SOCK_STREAM, 0)) < 0)
         {
-            perror ("socket()");
-            logg ("!%s: Can't create TCP socket\n", option);
+            logg ("!%s: Can't create TCP socket: %s\n", option, strerror(errno));
             optfree (opts);
             freeaddrinfo (res);
             return -1;
@@ -146,10 +145,9 @@
 
         if (connect (sockd, res->ai_addr, res->ai_addrlen) == -1)
         {
-            perror ("connect()");
+            logg ("!%s: Can't connect to clamd on %s:%s: %s\n", option,
+                  addr ? addr : "localhost", port, strerror(errno));
             closesocket (sockd);
-            logg ("!%s: Can't connect to clamd on %s:%s\n", option,
-                  addr ? addr : "localhost", port);
             optfree (opts);
             freeaddrinfo (res);
             return -1;
@@ -160,8 +158,7 @@
 
         if ((sockd = socket (AF_INET, SOCK_STREAM, 0)) < 0)
         {
-            perror ("socket()");
-            logg ("!%s: Can't create TCP socket\n", option);
+            logg ("!%s: Can't create TCP socket: %s\n", option, strerror(errno));
             optfree (opts);
             return -1;
         }
@@ -173,8 +170,8 @@
         {
             if ((he = gethostbyname (opt->strarg)) == 0)
             {
-                perror ("gethostbyname()");
-                logg ("^Clamd was NOT notified: Can't resolve hostname '%s'\n", opt->strarg);
+                logg ("^Clamd was NOT notified: Can't resolve hostname '%s': %s\n",
+                    opt->strarg, strerror(errno));
                 optfree (opts);
                 closesocket (sockd);
                 return -1;
@@ -189,9 +186,9 @@
             (sockd, (struct sockaddr *) &server2,
              sizeof (struct sockaddr_in)) < 0)
         {
-            perror ("connect()");
+            logg ("^Clamd was NOT notified: Can't connect to clamd on %s:%d: %s\n",
+                inet_ntoa (server2.sin_addr), ntohs (server2.sin_port), strerror(errno));
             closesocket (sockd);
-            logg ("^Clamd was NOT notified: Can't connect to clamd on %s:%d\n", inet_ntoa (server2.sin_addr), ntohs (server2.sin_port));
             optfree (opts);
             return -1;
         }
@@ -222,8 +219,7 @@
 
     if (sendln (sockd, "RELOAD", 7) < 0)
     {
-        perror ("send()");
-        logg ("!NotifyClamd: Could not write to clamd socket\n");
+        logg ("!NotifyClamd: Could not write to clamd socket: %s\n", strerror(errno));
         closesocket (sockd);
         return 1;
     }
