--- clamav-0.93rc1/configure.in~	2008-03-30 15:10:25.470753399 +0200
+++ clamav-0.93rc1/configure.in	2008-03-30 15:12:07.615636507 +0200
@@ -989,12 +989,12 @@
 dnl tcpwrappers support
 dnl rules from http://ma.ph-freiburg.de/tng/tng-technical/2002-01/msg00094.html
 AC_ARG_WITH([tcpwrappers],
-[  --with-tcpwrappers      support hosts.allow / hosts.deny (default=auto)],
+[  --with-tcpwrappers      support hosts.allow / hosts.deny (default=no)],
 [ case "$withval" in
   yes|no) tcpw="$withval" ;;
   *)      AC_MSG_ERROR([--with-tcpwrappers does not take an argument]) ;;
   esac],
-[ tcpw=auto ])
+[ tcpw=no ])
 
 if test "$tcpw" != "no" && test "$have_milter" = "yes" ; then
         AC_CHECK_HEADERS([tcpd.h],[
