--- clamav-0.101.0/Makefile.am~	2018-11-29 14:52:31.000000000 +0100
+++ clamav-0.101.0/Makefile.am	2018-12-07 09:00:16.238721757 +0100
@@ -34,7 +34,7 @@
 pkgconfigdir = $(libdir)/pkgconfig
 pkgconfig_DATA = libclamav.pc
 
-nodist_include_HEADERS = clamav-types.h
+pkginclude_HEADERS = clamav-types.h
 
 # don't complain that configuration files and databases are not removed, this is intended
 distuninstallcheck_listfiles = find . -type f ! -name clamd.conf ! -name freshclam.conf ! -name daily.cvd ! -name main.cvd -print
--- clamav-0.101.0/libclamav/Makefile.am~	2018-11-29 14:52:31.000000000 +0100
+++ clamav-0.101.0/libclamav/Makefile.am	2018-12-07 09:00:16.312058931 +0100
@@ -269,7 +269,7 @@
 libclamav_la_LDFLAGS += -Wl,@VERSIONSCRIPTFLAG@,@top_srcdir@/libclamav/libclamav.map
 endif
 
-include_HEADERS = clamav.h
+pkginclude_HEADERS = clamav.h
 
 libclamav_la_SOURCES = \
         matcher-ac.c \
--- clamav-0.101.0/libclamav.pc.in~	2018-11-29 14:52:31.000000000 +0100
+++ clamav-0.101.0/libclamav.pc.in	2018-12-10 11:11:47.748327272 +0100
@@ -1,7 +1,7 @@
 prefix=@prefix@
 exec_prefix=@exec_prefix@
 libdir=@libdir@
-includedir=@includedir@
+includedir=@includedir@/clamav
 
 Name: libclamav
 Description: A GPL virus scanner
--- clamav-0.101.0/clamav-config.in.orig	2018-11-29 14:52:31.000000000 +0100
+++ clamav-0.101.0/clamav-config.in	2018-12-10 19:32:41.185456107 +0100
@@ -3,7 +3,7 @@
 
 prefix=@prefix@
 exec_prefix=@exec_prefix@
-includedir=@includedir@
+includedir=@includedir@/clamav
 libdir=@libdir@
 
 usage()
@@ -55,7 +55,7 @@
 	;;
 
     --cflags)
-       	echo -I@includedir@ @CFLAGS@
+       	echo -I${includedir} @CFLAGS@
        	;;
 
     --libs)
