diff -urN clamav-0.12.org/clamscan/Makefile.am clamav-0.12/clamscan/Makefile.am
--- clamav-0.12.org/clamscan/Makefile.am	Thu May 23 13:50:34 2002
+++ clamav-0.12/clamscan/Makefile.am	Thu May 23 13:53:34 2002
@@ -41,18 +41,5 @@
     md5.h \
     threadarg.h
 
-CFLAGS = -O3 -Wall -DDATADIR=\"$(pkgdatadir)\"
+DEFS = @DEFS@ -DDATADIR=\"$(pkgdatadir)\"
 LIBS = @CLAM_LIBS@
-
-install-binPROGRAMS: $(bin_PROGRAMS)
-	@if test -x $(bin_PROGRAMS) -a ! -r clamscan.exe; then \
-	    strip $(bin_PROGRAMS); \
-	fi
-	@$(NORMAL_INSTALL)
-	$(mkinstalldirs) $(DESTDIR)$(bindir)
-	@list='$(bin_PROGRAMS)'; for p in $$list; do \
-	  if test -f $$p; then \
-	    echo "  $(INSTALL_PROGRAM) $$p $(DESTDIR)$(bindir)/`echo $$p|sed 's/$(EXEEXT)$$//'|sed '$(transform)'|sed 's/$$/$(EXEEXT)/'`"; \
-	     $(INSTALL_PROGRAM) $$p $(DESTDIR)$(bindir)/`echo $$p|sed 's/$(EXEEXT)$$//'|sed '$(transform)'|sed 's/$$/$(EXEEXT)/'`; \
-	  else :; fi; \
-	done
diff -urN clamav-0.12.org/database/Makefile.am clamav-0.12/database/Makefile.am
--- clamav-0.12.org/database/Makefile.am	Thu May 23 13:50:34 2002
+++ clamav-0.12/database/Makefile.am	Thu May 23 13:52:50 2002
@@ -18,16 +18,14 @@
 EXTRA_DIST = viruses.db
 
 install:
-	@mkdir -p $(pkgdatadir)
-	@chmod 711 $(pkgdatadir)
-	@cp -f viruses.db $(pkgdatadir)
-	@chmod 644 $(pkgdatadir)/viruses.db
-	@if test -r /etc/passwd; then \
-	    clamavuser=`cat /etc/passwd|grep clamav`; \
+	$(mkinstalldirs) $(DESTDIR)$(pkgdatadir)
+	@$(INSTALL_DATA) viruses.db $(DESTDIR)$(pkgdatadir)
+	@if test -r $(DESTDIR)/etc/passwd; then \
+	    clamavuser=`cat $(DESTDIR)/etc/passwd|grep clamav`; \
 	    if test -n "$$clamavuser"; then \
-		chown clamav $(pkgdatadir); \
-		chgrp clamav $(pkgdatadir); \
-		chown clamav $(pkgdatadir)/viruses.db; \
-		chgrp clamav $(pkgdatadir)/viruses.db; \
+		chown clamav $(DESTDIR)$(pkgdatadir); \
+		chgrp clamav $(DESTDIR)$(pkgdatadir); \
+		chown clamav $(DESTDIR)$(pkgdatadir)/viruses.db; \
+		chgrp clamav $(DESTDIR)$(pkgdatadir)/viruses.db; \
 	    fi; \
 	fi
diff -urN clamav-0.12.org/freshclam/Makefile.am clamav-0.12/freshclam/Makefile.am
--- clamav-0.12.org/freshclam/Makefile.am	Thu May 23 13:50:34 2002
+++ clamav-0.12/freshclam/Makefile.am	Thu May 23 13:54:22 2002
@@ -15,7 +15,7 @@
 #  along with this program; if not, write to the Free Software
 #  Foundation, Inc., 675 Mass Ave, Cambridge, MA 02139, USA.
 
-INCLUDES = -I../clamscan
+INCLUDES = -I$(top_srcdir)/clamscan
 
 bin_PROGRAMS = freshclam
 
@@ -29,7 +29,7 @@
     manager.h \
     shared.h
 
-LDADD = ../clamscan/md5.o ../clamscan/getopt.o
+LDADD = $(top_builddir)/clamscan/md5.o $(top_builddir)/clamscan/getopt.o
 
-CFLAGS = -O3 -Wall -DDATADIR=\"$(pkgdatadir)\"
+DEFS = @DEFS@ -DDATADIR=\"$(pkgdatadir)\"
 LIBS = @FRESH_LIBS@
