diff -urN clamav-0.80rc4.orig/etc/clamd.conf clamav-0.80rc4/etc/clamd.conf
--- clamav-0.80rc4.orig/etc/clamd.conf	2004-10-11 03:14:30.000000000 +0200
+++ clamav-0.80rc4/etc/clamd.conf	2004-10-16 23:14:10.722021576 +0200
@@ -5,13 +5,13 @@
 
 
 # Comment or remove the line below.
-Example
+#Example
 
 # Uncomment this option to enable logging.
 # LogFile must be writable for the user running daemon.
 # A full path is required.
 # Default: disabled
-#LogFile /tmp/clamd.log
+#LogFile /var/log/clamd.log
 
 # By default the log file is locked for writing - the lock protects against
 # running clamd multiple times (if want to run another clamd, please
@@ -40,7 +40,7 @@
 
 # Use system logger (can work together with LogFile).
 # Default: disabled
-#LogSyslog
+LogSyslog
 
 # Specify the type of syslog messages - please refer to 'man syslog'
 # for facility names.
@@ -54,7 +54,7 @@
 # This option allows you to save a process identifier of the listening
 # daemon (main thread).
 # Default: disabled
-#PidFile /var/run/clamd.pid
+PidFile /var/run/clamav/clamd.pid
 
 # Optional path to the global temporary directory.
 # Default: system specific (usually /tmp or /var/tmp).
@@ -69,7 +69,7 @@
 
 # Path to a local socket file the daemon will listen on.
 # Default: disabled
-LocalSocket /tmp/clamd
+LocalSocket /var/lib/clamav/clamd.socket
 
 # Remove stale socket after unclean shutdown.
 # Default: disabled
@@ -130,11 +130,11 @@
 
 # Run as a selected user (clamd must be started by root).
 # Default: disabled
-#User clamav
+User clamav
 
 # Initialize supplementary group access (clamd must be started by root).
 # Default: disabled
-#AllowSupplementaryGroups
+AllowSupplementaryGroups
 
 # Don't fork into background.
 # Default: disabled
@@ -273,15 +273,15 @@
 
 # Set access mask for Clamuko.
 # Default: disabled
-#ClamukoScanOnOpen
-#ClamukoScanOnClose
-#ClamukoScanOnExec
+ClamukoScanOnOpen
+ClamukoScanOnClose
+ClamukoScanOnExec
 
 # Set the include paths (all files in them will be scanned). You can have
 # multiple ClamukoIncludePath directives but each directory must be added
 # in a seperate line.
 # Default: disabled
-#ClamukoIncludePath /home
+ClamukoIncludePath /home
 #ClamukoIncludePath /students
 
 # Set the exclude paths. All subdirectories are also excluded.
diff -urN clamav-0.80rc4.orig/etc/freshclam.conf clamav-0.80rc4/etc/freshclam.conf
--- clamav-0.80rc4.orig/etc/freshclam.conf	2004-09-27 22:35:01.000000000 +0200
+++ clamav-0.80rc4/etc/freshclam.conf	2004-10-16 23:17:04.467608232 +0200
@@ -3,10 +3,14 @@
 ## Please read the freshclam.conf(5) manual before editing this file.
 ## This file may be optionally merged with clamd.conf.
 ##
-
+## PLD NOTE: Note that freshclam is called periodically via cron
+## Check /etc/sysconfig/clamd for details
+## Seems better to run task once per period than keep daemon running
+## only for that.
+## But if you have arguments please tell us.
 
 # Comment or remove the line below.
-Example
+#Example
 
 # Path to the database directory.
 # WARNING: It must match clamd.conf's directive!
@@ -19,11 +23,11 @@
 
 # Enable verbose logging.
 # Default: disabled
-#LogVerbose
+LogVerbose
 
 # Use system logger (can work together with UpdateLogFile).
 # Default: disabled
-#LogSyslog
+LogSyslog
 
 # Specify the type of syslog messages - please refer to 'man syslog'
 # for facility names.
@@ -37,7 +41,7 @@
 # By default when started freshclam drops privileges and switches to the
 # "clamav" user. This directive allows you to change the database owner.
 # Default: clamav (may depend on installation options)
-#DatabaseOwner clamav
+DatabaseOwner clamav
 
 # Use DNS to verify virus database version. Freshclam uses DNS TXT records
 # to verify database and software versions. We highly recommend enabling
@@ -62,7 +66,7 @@
 
 # Number of database checks per day.
 # Default: 12 (every two hours)
-#Checks 24
+Checks 2
 
 # Proxy settings
 # Default: disabled
@@ -76,7 +80,7 @@
 #NotifyClamd
 # By default it uses the hardcoded configuration file but you can force an
 # another one.
-#NotifyClamd /config/file/path
+NotifyClamd /etc/clamav.conf
 
 # Run command after successful database update.
 # Default: disabled
