--- clamav-0.96.2/configure.in~	2010-08-13 09:12:10.576795715 +0200
+++ clamav-0.96.2/configure.in	2010-08-13 09:43:56.526618279 +0200
@@ -1514,7 +1514,7 @@
 AC_LIB_FIND([ncurses], [ncurses/ncurses.h],
 	    AC_LANG_PROGRAM([#include <ncurses/ncurses.h>],
 			    [initscr(); KEY_RESIZE;]),
-	    [CURSES_CPPFLAGS="$INCNCURSES"; CURSES_LIBS="$LTLIBNCURSES";
+	    [CURSES_CPPFLAGS="$INCNCURSES"; CURSES_LIBS="$LTLIBNCURSES -ltinfo";
 	     CURSES_INCLUDE="<ncurses/ncurses.h>"],
 	    [])
 
