diff -Nur clamav-0.71-orig/etc/clamav.conf clamav-0.71/etc/clamav.conf
--- clamav-0.71-orig/etc/clamav.conf	Thu May 13 01:59:21 2004
+++ clamav-0.71/etc/clamav.conf	Thu May 20 00:04:51 2004
@@ -5,12 +5,12 @@
 
 
 # Comment or remove the line below.
-Example
+# Example
 
 # Uncomment this option to enable logging.
 # LogFile must be writable for the user running the daemon.
 # Full path is required.
-#LogFile /tmp/clamd.log
+#LogFile /var/log/clamd.log
 
 # By default the log file is locked for writing - the lock protects against
 # running clamd multiple times (if want to run another clamd, please
@@ -34,7 +34,7 @@
 #LogClean
 
 # Use system logger (can work together with LogFile).
-#LogSyslog
+LogSyslog
 
 # Specify the type of syslog messages - please refer to 'man syslog'
 # for facility names. Default is LOG_LOCAL6.
@@ -45,7 +45,7 @@
 
 # This option allows you to save the process identifier of the listening
 # daemon (main thread).
-#PidFile /var/run/clamd.pid
+PidFile /var/run/clamav/clamd.pid
 
 # Optional path to the global temporary directory.
 # Default is system specific - usually /var/tmp or /tmp.
@@ -62,10 +62,10 @@
 # Path to the local socket. The daemon doesn't change the mode of the
 # created file (portability reasons). You may want to create it in a directory
 # which is only accessible for a user running daemon.
-LocalSocket /tmp/clamd
+LocalSocket /var/lib/clamav/clamd.socket
 
 # Remove stale socket after unclean shutdown.
-#FixStaleSocket
+FixStaleSocket
 
 # TCP port address.
 #TCPSocket 3310
@@ -90,7 +90,7 @@
 # Maximal number of a threads running at the same time.
 # Default is 5, and it should be sufficient for a typical workstation.
 # You may need to increase threads number for a server machine.
-#MaxThreads 10
+MaxThreads 10
 
 # Waiting for data from a client socket will timeout after this time (seconds).
 # Default is 120. Value of 0 disables the timeout.
@@ -118,11 +118,11 @@
 
 # Run as selected user (clamd must be started by root).
 # By default it doesn't drop privileges.
-#User clamav
+User clamav
 
 # Initialize the supplementary group access (for all groups in /etc/group
 # user is added in. clamd must be started by root).
-#AllowSupplementaryGroups
+AllowSupplementaryGroups
 
 # Don't fork into background. Useful in debugging.
 #Foreground
@@ -145,7 +145,7 @@
 ##
 
 # Uncomment this option if you are planning to scan mail files.
-#ScanMail
+ScanMail
 
 ##
 ## Archive support
diff -Nur clamav-0.71-orig/etc/freshclam.conf clamav-0.71/etc/freshclam.conf
--- clamav-0.71-orig/etc/freshclam.conf	Tue May 11 01:19:48 2004
+++ clamav-0.71/etc/freshclam.conf	Thu May 20 00:03:30 2004
@@ -12,10 +12,10 @@
 #UpdateLogFile /var/log/freshclam.log
 
 # Enable verbose logging.
-#LogVerbose
+LogVerbose
 
 # Use system logger (can work together with UpdateLogFile).
-#LogSyslog
+LogSyslog
 
 # Specify the type of syslog messages - please refer to 'man syslog'
 # for facility names. Default is LOG_LOCAL6.
@@ -23,7 +23,7 @@
 
 # By default when freshclam is started by root it drops privileges and
 # switches to the "clamav" user. You can change this behaviour here.
-#DatabaseOwner clamav
+DatabaseOwner clamav
 
 # The main database mirror is database.clamav.net (this is a round-robin
 # DNS that points to many mirrors on the world) and in most cases you
@@ -35,7 +35,7 @@
 
 # How often check for a new database. We suggest checking for it every
 # two hours.
-Checks 12
+Checks 2
 
 # Proxy settings
 #HTTPProxyServer myproxy.com
@@ -45,6 +45,7 @@
 
 # Send the RELOAD command to clamd.
 #NotifyClamd [/optional/config/file/path]
+NotifyClamd /etc/clamav.conf
 
 # Run command after database update.
 #OnUpdateExecute command
