--- clamav-0.93rc1/configure.in~	2008-03-30 15:23:02.706953608 +0200
+++ clamav-0.93rc1/configure.in	2008-03-30 15:24:20.343998437 +0200
@@ -911,10 +911,7 @@
     dnl Check for libmilter and it's header files in the usual locations
     save_LDFLAGS="$LDFLAGS"
 	CLAMAV_MILTER_LIBS="$CLAMAV_MILTER_LIBS -lpthread"
-    if test -d /usr/lib/libmilter ; then
-	CLAMAV_MILTER_LIBS="$CLAMAV_MILTER_LIBS -L/usr/lib/libmilter"
-    fi
-    LDFLAGS="$LDFLAGS -lmilter $CLAMAV_MILTER_LIBS"
+    LDFLAGS="-lmilter $CLAMAV_MILTER_LIBS"
     AC_CHECK_LIB([milter],[mi_stop],[CLAMAV_MILTER_LIBS="-lmilter $CLAMAV_MILTER_LIBS"],[
 	dnl Older sendmails require libsm or libsmutil for support functions
 	AC_SEARCH_LIBS([strlcpy], [sm smutil], [test "$ac_cv_search_strlcpy" = "none required" || CLAMAV_MILTER_XLIB="$ac_cv_search_strlcpy"])
