--- clamav-0.93rc1/configure.in~	2008-03-30 15:22:36.772380455 +0200
+++ clamav-0.93rc1/configure.in	2008-03-30 15:22:22.568368090 +0200
@@ -307,8 +307,12 @@
   AC_DEFINE([CL_EXPERIMENTAL],1,[enable experimental code])
 fi
 
+AC_CHECK_FUNC(socket,, [
 AC_CHECK_LIB([socket], [bind], [LIBS="$LIBS -lsocket"; CLAMAV_MILTER_LIBS="$CLAMAV_MILTER_LIBS -lsocket"; FRESHCLAM_LIBS="$FRESHCLAM_LIBS -lsocket"; CLAMD_LIBS="$CLAMD_LIBS -lsocket"])
+])
+AC_CHECK_FUNC(gethostent,, [
 AC_SEARCH_LIBS([gethostent],[nsl], [(LIBS="$LIBS -lnsl"; CLAMAV_MILTER_LIBS="$CLAMAV_MILTER_LIBS -lnsl"; FRESHCLAM_LIBS="$FRESHCLAM_LIBS -lnsl"; CLAMD_LIBS="$CLAMD_LIBS -lnsl")])
+])
 
 AC_CHECK_FUNCS([poll setsid memcpy snprintf vsnprintf strerror_r strlcpy strlcat strcasestr inet_ntop setgroups initgroups ctime_r mkstemp])
 AC_FUNC_MMAP
