--- clamav-0.65.orig/libclamav/scanners.c	2003-11-09 20:14:42.000000000 +0100
+++ clamav-0.65/libclamav/scanners.c	2004-01-07 13:25:40.261486088 +0100
@@ -61,7 +61,7 @@
 #define DELIVERED_MAGIC_STR "Delivered-To: "
 #define BZIP_MAGIC_STR "BZh"
 
-#define ZIPOSDET 20 /* FIXME: Make it user definable */
+#define ZIPOSDET 200 /* FIXME: Make it user definable */
 
 int cli_magic_scandesc(int desc, char **virname, long int *scanned, const struct cl_node *root, const struct cl_limits *limits, int options, int *reclev);
 
