diff -urN clamav-0.66.org/etc/clamav.conf clamav-0.66/etc/clamav.conf
--- clamav-0.66.org/etc/clamav.conf	Wed Feb 11 11:51:14 2004
+++ clamav-0.66/etc/clamav.conf	Wed Feb 11 11:52:00 2004
@@ -5,12 +5,12 @@
 
 
 # Comment or remove the line below.
-Example
+# Example
 
 # Uncomment this option to enable logging.
 # LogFile must be writable for the user running the daemon.
 # Full path is required.
-#LogFile /tmp/clamd.log
+# LogFile /var/log/clamd.log
 
 # By default the log file is locked for writing - the lock protects against
 # running clamd multiple times (if want to run another clamd, please
@@ -34,14 +34,14 @@
 #LogClean
 
 # Use system logger (can work together with LogFile).
-#LogSyslog
+LogSyslog
 
 # Enable verbose logging.
 #LogVerbose
 
 # This option allows you to save the process identifier of the listening
 # daemon (main thread).
-#PidFile /var/run/clamd.pid
+PidFile /var/run/clamav/clamd.pid
 
 # Optional path to the global temporary directory.
 # Default is system specific - usually /var/tmp or /tmp.
@@ -58,10 +58,10 @@
 # Path to the local socket. The daemon doesn't change the mode of the
 # created file (portability reasons). You may want to create it in a directory
 # which is only accessible for a user running daemon.
-LocalSocket /tmp/clamd
+LocalSocket /var/lib/clamav/clamd.socket
 
 # Remove stale socket after unclean shutdown.
-#FixStaleSocket
+FixStaleSocket
 
 # TCP port address.
 #TCPSocket 3310
@@ -86,7 +86,7 @@
 # Maximal number of a threads running at the same time.
 # Default is 5, and it should be sufficient for a typical workstation.
 # You may need to increase threads number for a server machine.
-#MaxThreads 10
+MaxThreads 10
 
 # Thread (scanner - single task) will be stopped after this time (seconds).
 # Default is 180. Value of 0 disables the timeout. SECURITY HINT: Increase the
@@ -119,11 +119,11 @@
 
 # Run as selected user (clamd must be started by root).
 # By default it doesn't drop privileges.
-#User clamav
+User clamav
 
 # Initialize the supplementary group access (for all groups in /etc/group
 # user is added in. clamd must be started by root).
-#AllowSupplementaryGroups
+AllowSupplementaryGroups
 
 # Don't fork into background. Useful in debugging.
 #Foreground
@@ -136,7 +136,7 @@
 ##
 
 # Uncomment this option if you are planning to scan mail files.
-#ScanMail
+ScanMail
 
 ##
 ## Archive support
diff -urN clamav-0.66.org/etc/freshclam.conf clamav-0.66/etc/freshclam.conf
--- clamav-0.66.org/etc/freshclam.conf	Wed Feb 11 11:51:14 2004
+++ clamav-0.66/etc/freshclam.conf	Wed Feb 11 12:47:55 2004
@@ -3,20 +3,24 @@
 ## Please read the clamav.conf(5) manual before editing this file.
 ## This file may be optionally merged with clamav.conf.
 ##
-
+## PLD NOTE: Note that freshclam is called periodically via cron
+## Check /etc/sysconfig/clamd for details
+## Seems better to run task once per period than keep daemon running
+## only for that. 
+## But if you have arguments please tell us.
 
 # You can change the default database directory here.
 #DatabaseDirectory /var/lib/clamav
 
 # Path to the config file (make sure it has proper permissions)
-#UpdateLogFile /var/log/freshclam.log
+UpdateLogFile /var/log/freshclam.log
 
 # Enable verbose logging.
-#LogVerbose
+LogVerbose
 
 # By default when freshclam is started by root it drops privileges and
 # switches to the "clamav" user. You can change this behaviour here.
-#DatabaseOwner clamav
+DatabaseOwner clamav
 
 # The main database mirror is database.clamav.net (this is a round-robin
 # DNS that points to many mirrors on the world) and in most cases you
@@ -28,7 +32,7 @@
 
 # How often check for a new database. We suggest checking for it every
 # two hours.
-Checks 12
+Checks 2
 
 # Proxy settings
 #HTTPProxyServer myproxy.com
@@ -38,6 +42,7 @@
 
 # Send the RELOAD command to clamd.
 #NotifyClamd [/optional/config/file/path]
+NotifyClamd /etc/rc.d/init.d/clamd reload
 
 # Run command after database update.
 #OnUpdateExecute command
