--- clamav-0.100.1/libclamav/bytecode_vm.c~	2018-06-27 21:12:10.000000000 +0200
+++ clamav-0.100.1/libclamav/bytecode_vm.c	2018-07-26 10:48:10.316153596 +0200
@@ -1248,7 +1248,11 @@ int cli_vm_execute(const struct cli_bc *
         gettimeofday(&tv1, NULL);
         tv1.tv_sec -= tv0.tv_sec;
         tv1.tv_usec -= tv0.tv_usec;
+#ifdef __ILP32__
+        cli_dbgmsg("intepreter bytecode run finished in %lluus, after executing %u opcodes\n",
+#else
         cli_dbgmsg("interpreter bytecode run finished in %luus, after executing %u opcodes\n",
+#endif
                    tv1.tv_sec * 1000000 + tv1.tv_usec, pc);
     }
     if (stop == CL_EBYTECODE) {
--- clamav-0.104.1/libclamav/tomsfastmath/headers/tfm.h.orig	2021-11-02 20:09:25.000000000 +0100
+++ clamav-0.104.1/libclamav/tomsfastmath/headers/tfm.h	2021-12-11 10:43:55.832213985 +0100
@@ -118,7 +118,7 @@
 #endif
 
 /* autodetect x86-64 and make sure we are using 64-bit digits with x86-64 asm */
-#if defined(__x86_64__)
+#if defined(__x86_64__) && !defined(__ILP32__)
    #if defined(TFM_X86) || defined(TFM_SSE2) || defined(TFM_ARM)
        #error x86-64 detected, x86-32/SSE2/ARM optimizations are not valid!
    #endif
@@ -133,7 +133,7 @@
 #endif
 
 /* try to detect x86-32 */
-#if defined(__i386__) && !defined(TFM_SSE2)
+#if (defined(__i386__) || defined(__ILP32__)) && !defined(TFM_SSE2)
    #if defined(TFM_X86_64) || defined(TFM_ARM)
        #error x86-32 detected, x86-64/ARM optimizations are not valid!
    #endif
--- clamav-0.104.1/libclamav/tomsfastmath/mont/fp_montgomery_reduce.c.orig	2021-12-11 10:41:02.011008906 +0100
+++ clamav-0.104.1/libclamav/tomsfastmath/mont/fp_montgomery_reduce.c	2021-12-11 10:41:23.713556450 +0100
@@ -42,7 +42,7 @@ asm(
 : "%eax", "cc")
 
 /******************************************************************/
-#elif defined(TFM_X86_64)
+#elif defined(TFM_X86_64) && !defined(__ILP32__)
 /* x86-64 code */
 
 #define MONT_START

